crates/bevy_input/src/input.rs

```diff
@@ -3,9 +3,9 @@ use std::hash::Hash;
 
 /// A "press-able" input of type `T`
 pub struct Input<T> {
-    pressed: HashSet<T>,
-    just_pressed: HashSet<T>,
-    just_released: HashSet<T>,
+    pub pressed: HashSet<T>,
+    pub just_pressed: HashSet<T>,
+    pub just_released: HashSet<T>,
```

Comment:
#428 (not yet merged) adds methods that return iterators to these sets, which should remove the need for making these fields pub.

Reply:
This does look like a better idea - is there a way to merge pull requests, or should I just copy-paste the changes from there and do things that way?

Reply:
It's probably fine to leave this as is for now until cart is able to give his feedback/direction.
First we should see if he thinks iterator returning methods are the correct approach anyway :)

Reply:
You can't merge PRs exactly, but you can wait for one of the PRs to be merged into the main branch and then rebase your branch onto that change (i.e. rebase onto the latest main branch commits). You can technically rebase your branch onto the other branch in the meantime and set your PR's base branch to be the other branch, but this is probably confusing, and honestly, PRs that branch off of other PRs tend to become unwieldy quite fast.

Reply:
Ok, I'll wait thn